cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -365,3 +366,58 @@ def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
         from pcluster.cli_commands.dcv.connect import dcv_connect
 
         dcv_connect(args)
+
+
+class ExportClusterLogsCommand(CliCommand):
+    """Implement pcluster export-logs command."""
+
+    # CLI
+    name = "export-cluster-logs"
+    help = "Export the logs of the cluster."
+    description = help
+
+    def __init__(self, subparsers):
+        super().__init__(subparsers, name=self.name, help=self.help, description=self.description)
+
+    @staticmethod
+    def _regex_arg(value):
+        filter_regex = r"Name=(private-ip-address|start-time|end-time),Values=[\w\-_.,]+"
+        if not re.compile(fr"^({filter_regex})(\s+{filter_regex})*$").match(value):
+            raise argparse.ArgumentTypeError("filters parameter must be in the form Name=...,Values=... ")
+        return value
+
+    def register_command_args(self, parser: ArgumentParser) -> None:  # noqa: D102
+        parser.add_argument("cluster_name", help="Export the logs of the cluster name provided here.")
+        parser.add_argument("--output", help="Path to save log file archive to.", type=os.path.realpath)
+        # Export options
+        parser.add_argument("--bucket", required=True, help="s3 bucket to export cluster logs data to.")
```

Comment:
Should this mention that the bucket must be in the same region as the cluster?

Reply:
Added, thanks!